backend/src/test/java/com/shipping/backend/BackendApplicationTests.java

```diff
@@ -0,0 +1,54 @@
+package com.shipping.backend;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.shipping.backend.controllers.ShippingRetrivalServiceController;
+import com.shipping.backend.entities.BaseRequestMessage;
+import com.shipping.backend.entities.PackageTypeResponse;
+import com.shipping.backend.services.ShippingRequestSender;
+import com.shipping.backend.services.ShippingRequestSenderImpl;
+import com.shipping.backend.services.ShippingRetrivalService;
+import com.shipping.backend.services.ShippingRetrivalServiceImpl;
+import org.junit.Test;
+import org.springframework.amqp.core.AmqpTemplate;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+
+import java.io.IOException;
+import java.util.List;
+
+import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.mock;
+
+
+public class BackendApplicationTests {
```

Comment:
I'm getting confused on this test
What are you testing here? Controller? Service? Sender? All? o.o

Reply:
New unit test were implemented